<ui version="4.0" >
 <class>Graph</class>
 <widget class="QWidget" name="Graph" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>213</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>320</width>
    <height>213</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>320</width>
    <height>213</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="ui_xSelect" >
       <property name="font" >
        <font/>
       </property>
       <property name="styleSheet" >
        <string>background:rgb(30,255,30);</string>
       </property>
       <property name="editable" >
        <bool>false</bool>
       </property>
       <property name="currentIndex" >
        <number>1</number>
       </property>
       <property name="maxVisibleItems" >
        <number>11</number>
       </property>
       <item>
        <property name="text" >
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Acc X</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Acc Y</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Acc Z</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 0</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 1</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 2</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 3</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 4</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 5</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 6</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 7</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 8</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 9</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 10</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 11</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 12</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 13</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 14</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 15</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="ui_ySelect" >
       <property name="font" >
        <font/>
       </property>
       <property name="styleSheet" >
        <string>background:yellow;</string>
       </property>
       <property name="editable" >
        <bool>false</bool>
       </property>
       <property name="currentIndex" >
        <number>2</number>
       </property>
       <property name="maxVisibleItems" >
        <number>11</number>
       </property>
       <item>
        <property name="text" >
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Acc X</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Acc Y</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Acc Z</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 0</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 1</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 2</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 3</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 4</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 5</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 6</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 7</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 8</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 9</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 10</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 11</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 12</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 13</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 14</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 15</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="ui_zSelect" >
       <property name="font" >
        <font/>
       </property>
       <property name="styleSheet" >
        <string>background:red;</string>
       </property>
       <property name="editable" >
        <bool>false</bool>
       </property>
       <property name="currentIndex" >
        <number>3</number>
       </property>
       <property name="maxVisibleItems" >
        <number>11</number>
       </property>
       <item>
        <property name="text" >
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Acc X</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Acc Y</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Acc Z</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 0</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 1</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 2</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 3</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 4</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 5</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 6</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ang 7</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 8</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 9</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 10</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 11</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 12</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 13</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 14</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Dig 15</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" native="1" name="ui_graphWidget" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>100</horstretch>
         <verstretch>100</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>280</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>280</width>
         <height>160</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="ui_xValue" >
       <property name="minimumSize" >
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet" >
        <string>color:rgb(30,255,30);
font-size:12pt;
font:bold;</string>
       </property>
       <property name="text" >
        <string>9999</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="ui_yValue" >
       <property name="minimumSize" >
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet" >
        <string>color:yellow;
font-size:12pt;
font:bold;</string>
       </property>
       <property name="text" >
        <string>9999</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="ui_zValue" >
       <property name="minimumSize" >
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet" >
        <string>color:red;
font-size:12pt;
font:bold;</string>
       </property>
       <property name="text" >
        <string>9999</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="ui_delTimeLine" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="layoutDirection" >
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text" >
        <string>600</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
